### ISSUE TITLE
[Feature request] (Simpler) Rich embeds

### ISSUE THREAD

**Issue by sharifkhalladi** [zadam/trilium#886]
It would be useful to have the ability to include rich embeds without having to set up a code and render note and then using the "include note" functionality (I'm currently using this method to achieve rich media embeds, along with some minor CSS edits to hide the title of the included note).

I believe CKEditor has some native [media embed capabilities](https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html) but am unsure if we can simply enable these somehow.

I very much like how [Outline](https://github.com/outline/outline) approaches this, see an [example here](https://www.getoutline.com/integrations/airtable).

**Comment by zadam** [comment]
Yes, CKEditor has rich embed. I tried it a while ago and it did not work properly, but quite possibly it has been improved since. This would have to be enabled in the CKEditor build.

However I don't know if/how this would work with e.g. Airtable. I mean I don't know if all there is to integrate it is to enable this embedding and the rest will work automagically.

**Comment by sambbaron** [comment]
I am also interested in this.

https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html

**Comment by chee** [comment]
The media-embed library seems to be for embedding videos from websites like youtube. I'm primarily interested in embedding my own media, media that I have in another note. Similar to how you can see an image in the document it has been pasted into (it becomes an `<img>`), I'd like to be able to see included video notes become `<video/>` and audio become `<audio/>` without needing to create a render note

**Comment by zadam** [comment]
Hi, notes are normally added to another note through "include note" but this did not render a preview. I added it now so it looks like this:

![image](https://user-images.githubusercontent.com/617641/115126158-6534c000-9fcd-11eb-8883-a302e1ecf467.png)

**Comment by chee** [comment]
> Hi, notes are normally added to another note through [include note](https://github.com/zadam/trilium/wiki/Text-notes#include-note) but this did not render a preview. I added it now so it looks like this:
> 
> ![image](https://user-images.githubusercontent.com/617641/115126250-f0ae5100-9fcd-11eb-91db-c2aef99c7f33.png)

oh!! that's so exciting! a few of my notes are about to look so good!! thank you so much :green_heart:

**Comment by eliandoran** [comment]
Done in [d3cccae](https://github.com/TriliumNext/Trilium/commit/d3cccae2dfaef9deae5d22acac4324d7fc3d5526).
